addons/medical_feedback/XEH_postInit.sqf

```diff
@@ -21,21 +21,42 @@ GVAR(heartBeatEffectRunning) = false;
 ["ace_unconscious", {
     params ["_unit", "_unconscious"];
 
-    if (local _unit) then {
-        _unit setVariable ["tf_voiceVolume", [1, 0] select _unconscious, true];
-        _unit setVariable ["tf_unable_to_use_radio", _unconscious, true];
-        _unit setVariable ["acre_sys_core_isDisabled", _unconscious, true];
-    };
-
     if (_unit != ACE_player) exitWith {};
 
+    // Toggle unconscious player's ability to talk in radio addons
+    _unit setVariable ["tf_voiceVolume", [1, 0] select _unconscious, true];
+    _unit setVariable ["tf_unable_to_use_radio", _unconscious, true];
+    _unit setVariable ["acre_sys_core_isDisabled", _unconscious, true];
```

Comment:
@dedmen @jonpas 

Do ACRE/TFAR need these variables to be publicly set to prevent the local player from talking? I suspect the old code was only doing it to handle the potential for the player to change unit, but they're now updated whenever that happens anyway.

Reply:
voiceVolume yes.. sadly.
`tf_unable_to_use_radio` no. Local only

Reply:
Yeah, it appears it needs to be global.

Reply:
@dedmen In this PR I lower the hearing volume while unconscious, but muffled would be good to add.

and thanks guys!